src/algebra/category/Module/colimits.lean

```diff
@@ -3,11 +3,10 @@ Copyright (c) 2019 Scott Morrison. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Scott Morrison
 -/
-import algebra.category.Module.basic
-import group_theory.quotient_group
 import category_theory.limits.concrete_category
+import group_theory.quotient_group
 import category_theory.limits.shapes.kernels
-import category_theory.limits.shapes.concrete_category
+import algebra.category.Module.basic
```

Comment:
I thought this sorted imports. Is that not the case?

Reply:
The script still needs some hand-holding so some fixes are made by hand after those I sometimes sorted but maybe missed some